modules/core/src/main/java/org/apache/ignite/IgniteScheduler.java

```diff
@@ -40,11 +40,11 @@
  * to all nodes five times, once every minute, with initial delay of two seconds:
  * <pre name="code" class="java">
  * Ignition.ignite().scheduler().scheduleLocal(
- *     GridSchedulerFuture&lt;?&gt; = Ignition.ignite().scheduler().scheduleLocal(new Callable&lt;Object&gt;() {
+ *     SchedulerFuture&lt;?&gt; = Ignition.ignite().scheduler().scheduleLocal(new Callable&lt;Object&gt;() {
  *         &#64;Override public Object call() throws IgniteCheckedException {
  *             g.broadcast(new GridCallable() {...}).get();
  *         }
- *     }, "{2, 5} * * * * *" // 2 seconds delay with 5 executions only.
+ *     }, "{2, 5} * * * * * ?" // 2 seconds delay with 5 executions only.
```

Comment:
What does this question mark mean? Can you explain it in javadoc?

Reply:
Actually it is equivalent to '*' (no specific meaning except it is only allowed in the day of month and day of week fields), so I better to change it back to '*'.

Reply:
Also I fixed expression to satisfy description and add a line that seconds can be omitted.